mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -251,6 +259,27 @@ struct UpstreamSettings {
     subscriptions: Vec<Direction>,
 }
 
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct StorageSettings {
+    max_file_size: usize,
+    file_name: String,
```

Comment:
is it just a file name or a full path to a file? if later let's make it `file_path: PathBuf`

Reply:
As I suggested above, let's make it `folder_path` instead.

Reply:
Should be more clear now, I had started with file_name for a path. However, what I think works better is a directory. Then we can create however many files we want and guarantee they are bunched together